packages/core/src/http-middlewares/after/throw-for-stale-auth.js

```diff
@@ -6,12 +6,12 @@ const errors = require('../../errors');
  * The server-side will raise `RefreshAuthError` when `autoRefresh === true`.
  * Once we always run `throwForStatus` or a custom `afterResponse`, we can drop `throwForStaleAuth`.
  */
-const throwForStaleAuth = resp => {
-  if (resp.status === 401) {
-    throw new errors.ResponseError(resp);
+const throwForStaleAuth = response => {
+  if (response.status === 401) {
```

Comment:
No actually changes here, just renamed `resp` to `response` for clarity and to match what we use elsewhere.